### DISCUSSION TITLE
エラーがどうしても治らない時についてのTips

### DISCUSSION THREAD

**Discussion by soto0201** [jphacks/E_2202#80] [Ideas]
エラーがどうしても治らない時についてのTipsをフロントにて書いていきたいので、
皆さんがエラーが解決しない時に行っていることを書いていってくれると嬉しいです！

**Comment by soto0201** [comment]
- 編集するファイルを間違えていないか確かめてみましょう
	- 元のファイルをずっとコンパイルしている場合もあります

- 実行するファイルが正しいかを確かめてみましょう
	- 実行しているファイルがずっと古い場合もあります

- 一度ミニマムなアプリケーションを作ってみるのはいかがでしょう？

- 一度寝てみましょう
	- 疲れている場合、寝て起きたら解決している場合もあります

- ダッキングをしてみましょう
	- 誰かに説明するようにしながら状況を整理すると解決することが多いです

- デバッグ神社を建立してみましょう
	- 困ったときの神頼み

**Reply by soto0201** [reply]
気分展開に散歩してみましょう？

**Comment by Hamumayo55** [comment]
- 編集したファイルをセーブしましょう

**Comment by soto0201** [comment]
パソコンを再起動してみませんか？

**Comment by soto0201** [comment]
頼ることも技術です！解決に時間が掛かっている場合は有識者に聞いてみましょう
- その際はどういう手順でどのようなエラーが出るのか明確にするととても喜ばれます

**Comment by soto0201** [comment]
https://zenn.dev/yurukei20/articles/e9496c53797986
